src/app/loadout/plug-drawer/PlugSection.tsx

```diff
@@ -1,47 +1,76 @@
 import { PluggableInventoryItemDefinition } from 'app/inventory/item-types';
-import React from 'react';
+import _ from 'lodash';
+import React, { useCallback } from 'react';
 import styles from './PlugSection.m.scss';
 import SelectablePlug from './SelectablePlug';
 
+export interface PlugsWithMaxSelectable {
+  plugSetHash: number;
+  plugs: PluggableInventoryItemDefinition[];
+  maxSelectable: number;
+}
+
 export default function PlugSection({
-  plugs,
+  plugsWithMaxSelectable,
   selected,
   displayedStatHashes,
   isPlugSelectable,
   onPlugSelected,
   onPlugRemoved,
 }: {
-  /** A array of mods where plug.plugCategoryHash's are equal. */
-  plugs: readonly PluggableInventoryItemDefinition[];
+  plugsWithMaxSelectable: PlugsWithMaxSelectable;
   /** The current set of selected mods. Needed to figure out selection limits for some plugCategoryHashes. */
   selected: PluggableInventoryItemDefinition[];
   displayedStatHashes?: number[];
   isPlugSelectable(plug: PluggableInventoryItemDefinition): boolean;
-  onPlugSelected(mod: PluggableInventoryItemDefinition): void;
-  onPlugRemoved(mod: PluggableInventoryItemDefinition): void;
+  onPlugSelected(plugSetHash: number, mod: PluggableInventoryItemDefinition): void;
+  onPlugRemoved(plugSetHash: number, mod: PluggableInventoryItemDefinition): void;
 }) {
+  const { plugs, maxSelectable, plugSetHash } = plugsWithMaxSelectable;
+
+  const onPlugSelectedInternal = useCallback(
```

Comment:
The naming convention I'm moving towards is `onX` for event callback names, and `handleX` for event handler names.

Reply:
Ok can sort that out